manifests/0000_90_cluster-authentication-operator_02_servicemonitor.yaml

```diff
@@ -0,0 +1,49 @@
+# Configure cluster-monitoring for cluster-authentication-operator
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: openshift-authentication-operator
+  namespace: openshift-authenticaion-operator
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    metricRelabelings:
+    - action: drop
+      regex: etcd_(debugging|disk|request|server).*
+      sourceLabels:
+      - __name__
+    port: https
+    scheme: https
+    tlsConfig:
+      caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+      serverName: metrics.openshift-authentication-operator.svc
+  jobLabel: component
+  selector:
+    matchLabels:
+      app: openshift-authentication-operator
+---
+# Configure cluster-monitoring for cluster authentication resources
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: openshift-authentication
+  namespace: openshift-authentication
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    metricRelabelings:
+    - action: drop
+      regex: etcd_(debugging|disk|request|server).*
+      sourceLabels:
+      - __name__
+    port: https
```

Comment:
This should probably have the `/metrics` endpoint path specified?

Reply:
I think `/metrics` is the default, but doesn't hurt to be explicit 👍